embedded-tests/src/test/java/org/apache/druid/testing/embedded/query/SystemTableQueryTest.java

```diff
@@ -66,10 +53,9 @@ public void testSystemTableQueries_segmentsCount()
         dataSourceName1, dataSourceName2
     );
 
-    String[] result = cluster.callApi().runSql(query).split("\n");
-    Assertions.assertEquals(2, result.length);
-    Assertions.assertEquals(StringUtils.format("%s,10", dataSourceName1), result[0]);
-    Assertions.assertEquals(StringUtils.format("%s,10", dataSourceName2), result[1]);
+    String result = cluster.callApi().runSql(query);
+    Assertions.assertTrue(result.contains(dataSourceName1));
```

Comment:
This assertion doesn't really verify the count (as the test name suggests).
I think we can get rid of the `dataSourceName2` and just use `dataSourceName1` to simplify this test.